executor/src/lib.rs

```diff
@@ -1,6 +1,4 @@
 //! Tooling used for execution of compiled programs
 
-use powdr_ast::analyzed::Identity;
```

Comment:
Not sure why we had that?

Reply:
Maybe it used to have template parameters.